03_Chatter/01-creating-a-chat-app.markdown

```diff
@@ -31,7 +31,6 @@ There are a few more steps to get our app ready for the first start. First, we n
 
 ```bash
 cd chatter
-npm install
 mix ecto.create
```

Comment:
Do we need this anymore? I think phx.new by default will add the initial Ecto bits. At least, when I do run this, I am told `The database for Chatter.Repo has already been created`